src/Module/Admin/Themes/Index.php

```diff
@@ -37,7 +37,7 @@ protected function content(array $request = []): string
 
 		// reload active themes
 		if (!empty($_GET['action'])) {
-			self::checkFormSecurityTokenRedirectOnError(DI::baseUrl()->get() . '/admin/themes', 'admin_themes', 't');
+			self::checkFormSecurityTokenRedirectOnError('/admin/themes', 'admin_themes', 't');
```

Comment:
Why have you removed the base url here?

Reply:
It's applied automatically in `BaseModule::checkFormSecurityTokenRedirectOnError`, all the other calls aren't passing it for this reason. This is cleanup, only loosely related with the original issue.